### PR TITLE
STORE-3746 polishing

### PR DESCRIPTION
**Pull request by blaine-O**
Updates to the Custom Submission Form Hugo documentation per STORE-3746 polishing up

### CODE REVIEW

File: Hugo-StaticGen-Docs/content/applicationadmin/CustomSubmissionForms.md [hunk 1]
```diff
@@ -2,64 +2,83 @@
...
+  4. When you have finished the form and there are no more red checkmarks on the Mappings tab, save the form and close it.
+
+## 2. Verify the form and activate it
+
+a.  When you are finished with the form, from the main Custom Forms page, select the Form and then click on the **Verify** button.  This will prompt you to enter data to verify the validity of the form, that all of the required fields are able to be entered, and so forth.  
+
+b.  On the final page, **Review Submission** ensure that all of the sections have a green checkmark.  When finished click the **Verify Submission** button.
+
+{{% notice note %}}
+#### If you have all green checkmarks on the Review Submission page, but still get an error like the one below when Verifying,
```
Comment (pianomanfrazier) [inline Hugo-StaticGen-Docs/content/applicationadmin/CustomSubmissionForms.md]: image

This heading inside the notice box causes some problems.

File: Hugo-StaticGen-Docs/content/applicationadmin/CustomSubmissionForms.md [hunk 2]
```diff
@@ -2,64 +2,83 @@
...
+e.  Click **Submit for Approval**
+
+f.  Navigate to **Admin Tools &rarr; Data Management &rarr; Entries**  Sort by **Last Activity Date** descending to see the entry just entered at the top of the list.  Select the entry.
+
+g.  Click on **Approve.**
+
+h.  As a final check, go to the home page and **Search** for the new entry to ensure it is indexed and searchable.
+
+![CustomSubmission](/images/AppAdmin/CustomSubmission.JPG)
+
```
Comment (pianomanfrazier) [inline Hugo-StaticGen-Docs/content/applicationadmin/CustomSubmissionForms.md]: Remove extra line endings in this file.

File: Hugo-StaticGen-Docs/content/applicationadmin/CustomSubmissionForms.md (hunk 1 above)
Comment (blaine-O) [reply Hugo-StaticGen-Docs/content/applicationadmin/CustomSubmissionForms.md]: @pianomanfrazier Interesting, the ### heading?  Just want to make the text a little bit bigger, is there a better way to do that?

File: Hugo-StaticGen-Docs/content/applicationadmin/CustomSubmissionForms.md (hunk 2 above)
Comment (blaine-O) [reply Hugo-StaticGen-Docs/content/applicationadmin/CustomSubmissionForms.md]: @pianomanfrazier done

File: Hugo-StaticGen-Docs/content/applicationadmin/CustomSubmissionForms.md (hunk 1 above)
Comment (blaine-O) [reply Hugo-StaticGen-Docs/content/applicationadmin/CustomSubmissionForms.md]: @pianomanfrazier 
I did not see any other options in the markup, took it out.

Comment (blaine-O) [reply Hugo-StaticGen-Docs/content/applicationadmin/CustomSubmissionForms.md]: @pianomanfrazier Added **bold**

**Pull request merged**